files/scripts/swss.sh

```diff
@@ -242,6 +242,16 @@ function clean_up_chassis_db_tables()
 
 }
 
+is_feature_enabled()
+{
+    service=$1
```

Comment:
There's an issue with this code. the $1 is passed in from DEPENDENT list. This could be something like this macsec@0. Needs to parse out the dev part. 
recommend code like this:  service=$1 changes to:
s=$1
service=${s%&*}
This will make config_db query to get the proper record.